src/linalg/rankUpdate.jl

```diff
@@ -61,6 +61,27 @@ function rankUpdate!(C::HermOrSym{T,Diagonal{T,Vector{T}}}, A::StridedMatrix{T},
     C
 end
 
+function rankUpdate!(C::HermOrSym{T,UniformBlockDiagonal{T}}, A::StridedMatrix{T}, α, β) where {T,S}
+    Cdat = C.data.data
+    @. Cdat = β * Cdat
+    nblks = size(Cdat, 3)
+    blksize = size(Cdat, 1)
+
+    for k in 1:nblks
+        ioffset = (k - 1) * blksize
+        joffset = (k - 1) * blksize
+        for i in 1:blksize, j in 1:blksize
```

Comment:
```suggestion
        for i in 1:blksize, j in 1:i
```
Only need to update the lower triangle of `C`.

Reply:
Oh, good trick. I wasn't even thinking that it's not just `UniformBlockDiagonal`, but also `Symmetric`.